Main.cs

```diff
@@ -69,8 +69,6 @@ protected override void Update(GameTime gameTime)
                 gamePlay.Update();
             }
 
-            gamePlay.Update();
```

Comment:
this was the pause bug -- should have been moved up into one of those ifs and totally forgot to remove it instead. explains the double speed of the mobs and pause being ignored. all is well now! Thank you VS debugger 😄